.github/workflows/pull-request-account-compression.yml

```
@@ -0,0 +1,89 @@
+name: Account Compression Pull Request
+
+on:
+  pull_request:
+    paths:
+      - "account-compression/**"
+      - "ci/*-version.sh"
+  push:
+    branches: [master]
+    paths:
+      - "account-compression/**"
+      - "ci/*-version.sh"
+
+jobs:
+  cargo-test-sbf:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set env vars
+        run: |
+          source ci/rust-version.sh
+          echo "RUST_STABLE=$rust_stable" >> $GITHUB_ENV
+          source ci/solana-version.sh
+          echo "SOLANA_VERSION=$solana_version" >> $GITHUB_ENV
+
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: ${{ env.RUST_STABLE }}
+          override: true
+          profile: minimal
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+          key: cargo-build-${{ hashFiles('**/Cargo.lock') }}-${{ env.RUST_STABLE}}
+
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin/rustfilt
+          key: cargo-sbf-bins-${{ runner.os }}
+
+      - uses: actions/cache@v2
+        with:
+          path: ~/.cache/solana
+          key: solana-${{ env.SOLANA_VERSION }}
+
+      - name: Install dependencies
+        run: |
+          ./ci/install-build-deps.sh
+          ./ci/install-program-deps.sh
+          echo "$HOME/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
+
+      - name: Build and test
+        run: ./ci/cargo-test-sbf.sh
```

This will blow out the disk space on the runner unless you specify exactly which programs to build:
```suggestion
        run: ./ci/cargo-test-sbf.sh account-compression/programs/account-compression
```
and in another step
```
      - name: Build and test
        run: ./ci/cargo-test-sbf.sh account-compression/programs/noop
```
Unless you're better off calling the anchor version of that from within the correct directory

---

I think I should call the correct anchor version from within the correct directory, since using different anchor clis produces different versions of the binary 🙈

---

ah, then yeah you don't need it, just `anchor build` is fine